app/models/manageiq/providers/openstack/network_manager/floating_ip.rb

```diff
@@ -1,2 +1,105 @@
 class ManageIQ::Providers::Openstack::NetworkManager::FloatingIp < ::FloatingIp
+  include ProviderObjectMixin
+  include AsyncDeleteMixin
+  include SupportsFeatureMixin
+
+  supports :create_floating_ip
```

Comment:
its the same as here https://github.com/ManageIQ/manageiq/pull/11751#pullrequestreview-5594569
keep in mind that this is mostly for the UI part to check if they can show a button or not.
So the UI would ask a Manager if it can `create_floating_ip` and ask a FloatingIp if it can `update`

Add `update` and `delete` here. Or even better in the base class with a check for the expected methods that are being called.

Add `create_floating_ip` to `NetworkManager`